english/forest/shelter/play-dead/get-out/get-out.md

```diff
@@ -11,8 +11,8 @@ Still dazzled and half deafened by the lighting,
 you see a figure on a distant hill dancing wildly under the storm rains. 
 You think it could be a shawman.
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.